### PR TITLE
Add URI transform support

### PR DESCRIPTION
**Pull request by theikkila**

### CODE REVIEW

File: src/spec_tools/transform.cljc [hunk 1]
```diff
@@ -69,6 +70,15 @@
       :else x)
     x))
 
+(defn string->uri [_ x]
+  (if (string? x)
+    (try
+      #?(:clj  (URI/create x)
+         ;; https://stackoverflow.com/questions/11528249/uri-validation-in-javascript
```
Comment (Deraen) [inline src/spec_tools/transform.cljc]: Closure library provides URI class:

https://google.github.io/closure-library/api/goog.Uri.html

**Comment by miikka** [comment]
The build is failing because of Travis issues, fixed in #195. I ran the tests in this branch locally and they pass, so that's good.

File: src/spec_tools/transform.cljc (hunk 1 above)
Comment (theikkila) [reply src/spec_tools/transform.cljc]: I'll change cljs version to use that. Thanks!

**Comment by ikitommi** [comment]
Thanks!

**Pull request merged**